services/configuration/src/cmd.rs

```diff
@@ -0,0 +1,215 @@
+// Copyright 2018 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
I find it really disturbing that need _all this boilerplate_ for a single command-line parameter. Is it really necessary? All these unrelated defaults like testnet configuration, addresses, etc. Can we maybe reuse something between services?

Reply:
Boilerplate will be reduced in the future PRs as it requires refactoring and improving of Finalize cmd and testnet generation.